Ghidra/Features/Base/src/main/java/ghidra/base/help/GhidraHelpService.java

```diff
@@ -107,23 +109,19 @@ private Map<ResourceFile, Set<URL>> findHelpSetsByModule() {
 
 	private Map<ResourceFile, Set<URL>> mapHelpToModule(Collection<ResourceFile> moduleRoots,
 			Set<URL> allHelpSets) {
-
 		Map<ResourceFile, Set<URL>> results = new HashMap<>();
+		Spliterator<URL> spliterator = Spliterators.spliterator(
+				allHelpSets.iterator(),
+				allHelpSets.size(),
+				Spliterator.ORDERED
+		);
```

Comment:
Any particular reason for not using `allHelpSets.stream()`?

Reply:
Whoops! Didn't think about that since I was basing off of the original code (thought the original code had a reason to want to use `Iterator<T>`). I like this much more. I'll change to this solution commit after testing!